packages/winpanda/extra/src/winpanda/tests/unit/test_core_rc_ctx.py

```diff
@@ -0,0 +1,94 @@
+import unittest
+import mock
+
+from common.storage import ISTOR_NODE
+from core.rc_ctx import ResourceContext, RCCONTEXT_ITEM
+
+STUB_ISTORE_NODES_CONFIG = {
+    ISTOR_NODE.ROOT: 'val_root',
+    ISTOR_NODE.TMP: 'val_tmp',
+    ISTOR_NODE.BIN: 'val_bin',
+    ISTOR_NODE.LIB: 'val_lib',
+    ISTOR_NODE.CFG: mock.Mock(),
+    ISTOR_NODE.WORK: mock.Mock(),
+    ISTOR_NODE.RUN: mock.Mock(),
+    ISTOR_NODE.LOG: mock.Mock(),
+    ISTOR_NODE.PKGREPO: mock.Mock()
+}
+
+RCCONTEXT_ITEMS = [
+    RCCONTEXT_ITEM.DCOS_INST_DPATH,
+    RCCONTEXT_ITEM.DCOS_CFG_DPATH,
+    RCCONTEXT_ITEM.DCOS_WORK_DPATH,
+    RCCONTEXT_ITEM.DCOS_RUN_DPATH,
+    RCCONTEXT_ITEM.DCOS_LOG_DPATH,
+    RCCONTEXT_ITEM.DCOS_TMP_DPATH,
+    RCCONTEXT_ITEM.DCOS_BIN_DPATH,
+    RCCONTEXT_ITEM.DCOS_LIB_DPATH,
+    RCCONTEXT_ITEM.PKG_INST_DPATH,
+    RCCONTEXT_ITEM.PKG_LOG_DPATH,
+    RCCONTEXT_ITEM.PKG_RTD_DPATH,
+    RCCONTEXT_ITEM.PKG_WORK_DPATH,
+    RCCONTEXT_ITEM.PKG_SHRCFG_DPATH
+]
+
+
+class TestPackageId(unittest.TestCase):
+    @staticmethod
+    def get_nodes():
+        return type("Foo", (object,), STUB_ISTORE_NODES_CONFIG)()
+
+    def test_default_get_items_should_be_empty(self):
+        context = ResourceContext()
+
+        assert context.get_items() == {}
+
+    def test_cluster_conf_items_should_correspond_stub(self):
+        context = ResourceContext(istor_nodes=self.get_nodes())
+        items = context.get_items()
+
+        assert items[RCCONTEXT_ITEM.DCOS_INST_DPATH] == 'val_root'
+        assert items[RCCONTEXT_ITEM.DCOS_TMP_DPATH] == 'val_tmp'
+        assert items[RCCONTEXT_ITEM.DCOS_BIN_DPATH] == 'val_bin'
+        assert items[RCCONTEXT_ITEM.DCOS_LIB_DPATH] == 'val_lib'
+
+    def test_cluster_items_should_be_not_empty(self):
```

Comment:
This test can raise an error.

Reply:
it is just checking default configuration, could you please describe more details how error can happend?